modules/tinymce/src/core/test/ts/browser/undo/UndoKeyboardShortcutsTest.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyAssertions, TinyContentActions } from '@ephox/mcagar';
+import { PlatformDetection } from '@ephox/sand';
+import { TinyHooks } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.undo.UndoKeyboardShortcutTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [], true);
+  const platform = PlatformDetection.detect();
+
+  // idk why but it works, seperate test command for macOS, Firefox, and the rest
+  const selectAll = (editor: Editor) => {
+    editor.execCommand('SelectAll');
+  };
+  const Undo = (editor: Editor) => {
+    TinyContentActions.keystroke(editor, 'Z'.charCodeAt(0), platform.os.isMacOS() ? { metaKey: true } : { ctrl: true });
+
+  };
+  const Redo = (editor: Editor) => {
+    TinyContentActions.keystroke(editor, 'Y'.charCodeAt(0), platform.os.isMacOS() ? { metaKey: true } : { ctrl: true });
+
+  };
+  const Delete = (editor: Editor) => {
+    editor.execCommand('Delete');
+  };
```

Comment:
These names also need fixing, as they should be camel case.

Reply:
https://github.com/tinymce/tinymce/pull/7533/commits/efb852d6a10c3d8ed83be6b0911249e21ed2c88e